plugins/woocommerce/tests/e2e-pw/tests/product/block-editor/product-inventory-block-editor.spec.js

```diff
@@ -245,7 +245,6 @@ test(
 			// Verify image in store frontend
 			await page.goto( product.permalink );
 
-			await page.locator( 'button[name="add-to-cart"]' ).click();
 			await page.locator( 'button[name="add-to-cart"]' ).click();
```

Comment:
Clicking twice isn't needed because in the previous `step` of this test we already added this product to the cart, so one single click is enough to trigger the error.